test/NuGet.Tests.Apex/NuGet.Tests.Apex/Utility/CommonUtility.cs

```diff
@@ -458,13 +458,13 @@ public static void WaitForFileNotExists(FileInfo file)
                 $"{file.FullName} still existed after {Timeout}.");
         }
 
-        public static void WaitForDirectoryExists(string directoryPath)
+        public static void AssertDirectoryExists(string directoryPath, bool isExists)
         {
             Omni.Common.WaitFor.IsTrue(
-                () => !Directory.Exists(directoryPath),
+                () => isExists ? Directory.Exists(directoryPath) : !Directory.Exists(directoryPath),
```

Comment:
I'm not sure I understand this. Would it just return true if the directory already exists?

Reply:
This “ternary conditional operator” is used to verify that the directory path exists, because some cases we verify that it exists and some cases we verify that it doesn't exist, so we indicate whether we expect the path to exist or not by the parameter `isExists`. 

If you think it's not clear, we'll write this validation separately.

Reply:
I think it's easier to just validate it separately.

Reply:
Hi @nkolev92, I have updated this, could you please help review and approve it, thanks!